src/main/java/com/google/sps/utils/Game.java

```diff
@@ -0,0 +1,66 @@
+package com.google.sps.utils;
+import com.google.appengine.api.datastore.Entity;
+import java.lang.IllegalArgumentException;
+import java.util.List;
+import java.util.ArrayList;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+
+public final class Game {
+    DatastoreService datastore;
+    private Entity entity;
+
+    public Game() {
```

Comment:
Oh, one more thing. I recommend following the same pattern as for the `User` class -- take the `Entity` as a parameter to the constructor. That way you can create a `Game` from an `Entity` retrieved from datastore.

(Checking that `entity.getKind().equals("Game")` is also a good idea)

Reply:
Okay I added a constructor for this